client/client.go

```diff
@@ -544,3 +544,15 @@ func getEventsResult(res *access.EventsResponse) ([]BlockEvents, error) {
 
 	return results, nil
 }
+
```

Comment:
have the access node implementations already been done for this changes?

Reply:
It's in review https://github.com/onflow/flow-go/pull/377